### PR TITLE
Patch release 1.0.1

### PR DESCRIPTION
**Pull request by itrujnara**
Fixed bugs that caused crashes, removed some unnecessary file publishing

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/reportho/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/reportho _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.0.1](https://github.com/nf-core/reportho/releases/tag/1.0.1) [2024-06-11]
```
Comment (JoseEspinosa) [inline CHANGELOG.md]: Remember to update if not released today
```suggestion
## [v1.0.1](https://github.com/nf-core/reportho/releases/tag/1.0.1) [2024-06-14]
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM, just a small tyop corrected (c/p error likely)

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
+
+### `Fixed`
+
+- Fixed minor bugs that caused compilation errors in the structural alignment section
```
Comment (mashehu) [inline CHANGELOG.md:10]: any way to catch these kind of things with different parameter matrices in the CI tests?

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Also, seems that test are failing, please fix the issue before merging

File: CHANGELOG.md (hunk 2 above)
Comment (itrujnara) [reply CHANGELOG.md:10]: Definitely, I can create a test for that. However, that means quite a lot of extra computation at each CI run. What do you say?

Comment (itrujnara) [reply CHANGELOG.md:10]: @mashehu

Comment (JoseEspinosa) [reply CHANGELOG.md:10]: Would be nice if we could move this forward to make available the output of the pipeline into the nf-core web

Comment (mashehu) [reply CHANGELOG.md:10]: okay for me if the tests are actually too computationally expensive

**Review by mashehu — APPROVED** [review]

**Review by JoseEspinosa — APPROVED** [review]
The full test have successfully run so I think it is time to release the patch 🎉

**Pull request merged**